coderd/httpapi/httpapi.go

```diff
@@ -154,6 +154,7 @@ func ResourceNotFound(rw http.ResponseWriter) {
 func Forbidden(rw http.ResponseWriter) {
 	Write(context.Background(), rw, http.StatusForbidden, codersdk.Response{
 		Message: "Forbidden.",
+		Detail:  "You don't have permission to view this page. If you believe this is a mistake, please contact your administrator or try signing in with different credentials.",
```

Comment:
```suggestion
		Detail:  "You don't have permission to view this content. If you believe this is a mistake, please contact your administrator or try signing in with different credentials.",
```

thinking of this in the context of using the CLI or the API directly, "page" doesn't make a ton of sense